src/ring_theory/power_series/basic.lean

```diff
@@ -780,10 +780,22 @@ by simp only [coeff_coe, mv_power_series.coeff_mul, coeff_mul]
   ((C a : mv_polynomial σ R) : mv_power_series σ R) = mv_power_series.C σ R a :=
 coe_monomial _ _
 
+@[simp, norm_cast] lemma coe_bit0 :
+  ((bit0 φ : mv_polynomial σ R) : mv_power_series σ R) = bit0 (φ : mv_power_series σ R) :=
+coe_add _ _
+
+@[simp, norm_cast] lemma coe_bit1 :
+  ((bit1 φ : mv_polynomial σ R) : mv_power_series σ R) = bit1 (φ : mv_power_series σ R) :=
+by rw [bit1, bit1, coe_add, coe_one, coe_bit0]
+
 @[simp, norm_cast] lemma coe_X (s : σ) :
   ((X s : mv_polynomial σ R) : mv_power_series σ R) = mv_power_series.X s :=
 coe_monomial _ _
 
+@[simp, norm_cast] lemma coe_X_pow (n : ℕ) (i : σ) :
+  ((X i ^ n : mv_polynomial σ R) : mv_power_series σ R) = mv_power_series.X i ^ n :=
+by { rw [X_pow_eq_monomial, mv_power_series.X_pow_eq], exact coe_monomial _ 1 }
```

Comment:
Is this true more generally as `coe_pow`?

Reply:
I think the answer is yes, and the proof is `coe_to_mv_power_series.ring_hom.map_pow`

Reply:
Even better! thanks